src/components/QueryWrapper/index.tsx

```diff
@@ -3,10 +3,16 @@ import Loader from "src/components/Loader";
 
 type PropsType = {
   query: QueryResult;
-  errorMessage: string;
+  errorMessage?: string;
 };
 
-const QueryWrapper: React.FC<React.PropsWithChildren<PropsType>> = ({ query, errorMessage, children }) => {
+const DEFAULT_ERROR = "Something happened...";
+
+const QueryWrapper: React.FC<React.PropsWithChildren<PropsType>> = ({
+  query,
+  errorMessage = DEFAULT_ERROR,
```

Comment:
An easier and more consistent way to do things might be to just throw an error if there is no behavior specified. Doing this should trigger the ErrorBoundary in App/index.tsx and display its generic error message.

Reply:
On it !